css/style.css

```diff
@@ -1,4 +1,4 @@
-#navbar{
+#navbar {
```

Comment:
- [OPTIONAL] Always state the `WHY` you ask the students to make some changes to their line or block  of code. With this, it would enable the students to learn very quickly and give them insight of why they should implement your requests.